frontend/src/core/api/types.js

```diff
@@ -55,8 +55,10 @@ export type OtherLocaleTranslation = {|
  * Translation that comes from a machine (Machine Translation,
  * Translation Memory... ).
  */
+type SourceType = 'translation-memory' | 'google-translate' | 'microsoft-translator' | 'microsoft-terminology' | 'transvision' | 'caighdean'
```

Comment:
Can you split this into several lines, to avoid being longer than 100 characters? Here's an example of how we do it elsewhere: https://github.com/mozilla/pontoon/blob/master/frontend/src/modules/batchactions/reducer.js#L25-L33

Reply:
Ooo definitely looks neater. Thanks for pointing it out!